src/windows/dialog_choice_window.c

```diff
@@ -4,6 +4,8 @@
 
 #include "dialog_choice_window.h"
 
+#define MARGIN 10
```

Comment:
Any reason this is defined here, while other macros are defined in the associated .h files?